### PR TITLE
Schemas roles, users

### PR DESCRIPTION
**Pull request by pisaychuk**
Currently we intensively work on updating api docs with request/response schemas etc.

 I added these to api doc, added schemas for request/response sections as well:

GET /identitydomains/{domainName}/{roles}
GET /identitydomains/{domainName}/{roles}/{roleId}
GET /identitydomains/{domainName}/users
GET, PUT, POST /identitydomains/{domainName}/users/{userId}

### CODE REVIEW

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -20404,269 +20404,625 @@
...
-                    "NPANXX": {
-                      "type": "string"
-                    },
-                    "Tier": {
-                      "type": "integer"
-                    },
-                    "Count": {
-                      "type": "integer"
-                    }
+    "/identitydomains/{domainName}/{roles}": {
```
Comment (tmonck) [inline site/specs-temp/numbers.json:20408]: So looking at the response are we sure we want to expose this out to the public? The server side isn't filtering out roles that we do not mark as "CustomerVisible" which means that no roles would be non-customer visible if they are consuming the API. We would also be exposing which of our Roles are "Internally" scoped. 

We might want to get Infosec to weight in on some of the information that we are documenting and exposing to the customers.

Comment (pisaychuk) [reply site/specs-temp/numbers.json:20408]: Hey @tmonck, I think backend filters out all roles user should not have access to. Depending on user privileges one will see both internal and external, another will see only external / owned by this user.

Comment (pisaychuk) [reply site/specs-temp/numbers.json:20408]: Screenshot 2022-10-05 at 16 15 32

**Comment by mrmeyers99** [comment]
@tmonck You don't want to merge these changes at this point right?  I think I'll go ahead and close this for now.

**Comment by pisaychuk** [comment]
closing it for now, will reopen once we have new api doc repo created

**Pull request closed without merging**